apps/angular/injection-token/src/app/video.component.ts

```diff
@@ -1,14 +1,22 @@
-import { Component } from '@angular/core';
+import { Component, inject } from '@angular/core';
 import { TimerContainerComponent } from './timer-container.component';
+import { DEFAULT_TIMER_TOKEN } from './data';
 
 @Component({
   selector: 'app-video',
   standalone: true,
   imports: [TimerContainerComponent],
+  providers: [
+    {
+      provide: DEFAULT_TIMER_TOKEN,
+      useValue: '1000'
+    }
+  ], 
```

Comment:
there is a better solution. You can provide a default value to your injection token.

Reply:
@tomalaforge  mybad i should have removed it , since the injection_tokens default value is already 1000

Reply:
no you don't have any default value

Reply:
@tomalaforge  the Timer token from the data.ts  is injected in timerContainer component ,  and the video component will have no need for providing '1000' value, since its already injected in timerContainer component

Reply:
I don't understand what your are trying to say. The goal is to have a default value at 1000 and override it when needed. If you delete this provider, you will have an error

Reply:
@tomalaforge  i have realized the syntax for setting default value in data.ts was wrong,
but ya after fixing that, i have to remove the provider from video compoent since it have same timer as the default value, just need to inject the injection_token in that component